.gitignore

```diff
@@ -59,3 +59,6 @@ gradle-app.setting
 
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
 # gradle/wrapper/gradle-wrapper.properties
+
+# Hibernate application properties file
+application.properties
```

Comment:
That's cool. I all the time was commiting properties but your way with .sample looks better!